build.gradle

```diff
@@ -37,9 +37,9 @@ dependencies {
   compile gradleApi()
   compile localGroovy()
 
-  compile 'com.squareup.spoon:spoon-runner:1.1.2'
+  compile 'com.squareup.spoon:spoon-runner:1.1.5'
 
-  compile 'com.android.tools.build:gradle:0.14.1'
+  compile 'com.android.tools.build:gradle:1.0.0'
```

Comment:
It could be a conflict with guava when you run a lint task from gradle (for example, with `./gradlew check`).
https://github.com/stanfy/spoon-gradle-plugin/issues/33